backend/backend/main.py

```diff
@@ -33,3 +34,6 @@ async def lifespan(app: FastAPI):
 
 # Include API routes
 app.include_router(api_router)
+
+# Mount Gradio routes
+app = gr.mount_gradio_app(app, gradio_routes, path="/gradio")
```

Comment:
Consider separating the API and Gradio routes into separate files for better organization.